grabthar.yml

```diff
@@ -2,6 +2,9 @@ entrypoint: ./main.ts
 exclude:
   - bin
   - packages
+target: es2015
+build:
+  minify: false
```

Comment:
Minification can introduce some stuff that QJSEngine doesn't like. But the primary motivation is that QJSEngine doesn't support source maps.

There should probably be a TODO to have QJSEngine print out its bundle, to improve debugging if someone files a bug report.